assets/app/view/log.rb

```diff
@@ -5,18 +5,16 @@ class Log < Snabberb::Component
     needs :log
 
     def render
-      reverse_scroll = lambda do |event|
-        %x{
-          var e = #{event}
-          e.preventDefault()
-          e.currentTarget.scrollTop -= e.deltaY
-        }
+      scroll_to_bottom = lambda do |vnode|
+        elm = Native(vnode)['elm']
+        elm.scrollTop = elm.scrollHeight
       end
 
       props = {
-        on: { wheel: reverse_scroll },
+        hook: {
```

Comment:
wow ok that seems simple if it works

can you add key: i think you generally need that if you're doing hooks

does this scroll to the bottom anytime we patch though? what happens if you're scrolling and you get a new event?

Reply:
for example, if you look at the logs and then someone sends a chat,

Reply:
If you're looking at the logs and then someone sends a chat, it does scroll to accommodate the new message, but it only really works if you're scrolled to the bottom of chat. Otherwise, it effectively scrolls down by the amount of the new message.

If you're scrolled up in the log and a new message appears, it should either stay in place or scroll to bottom, but the existing behavior isn't ideal.

I also discovered that this doesn't actually fix #346.  I'm going to close the PR for now and re-open when I find a more complete solution.

Reply:
Yes, it scrolls to the bottom any time we patch. If you're scrolling and you get a new event, it scrolls to the bottom. (Existing behavior on master is that a new event scrolls the window by the height of the new event.)